src/WarcraftLegacies.Source/Quests/Frostwolf/QuestHighmountain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.QuestSystem;
+using static War3Api.Common;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.LegendSystem;
+
+namespace WarcraftLegacies.Source.Quests.Frostwolf
+{
+  /// <summary>
+  /// Bring any hero to the Echo Isles to unlock the base.
+  /// </summary>
+  public sealed class QuestHighmountain : QuestData
+  {
+    private readonly LegendaryHero _cairne;
+
+    /// <inheritdoc />
+    public QuestHighmountain(LegendaryHero cairne) : base("Invitation to a Feast",
+      "The Taurens of Highmountain have been isolated for centuries. As a gesture of reconciliation, Cairne has set off to offer them an invitation to a feast in Thunderbluff.",
```

Comment:
I think we don't want to imply that he _has_ set off, because the player may simply choose not to do this. "Reconciliation" implies previous hostilities so let's avoid that too.
```suggestion
      "Scouts report sighting of the Highmountain totem, thought lost long ago when the Broken Isles were shattered. As a gesture of renewed welcome, Cairne might offer them an invitation to a feast in Thunderbluff.",
```